docs/api.md

```diff
@@ -143,31 +150,53 @@ const FocusableNode = () => (
+The `SpatialNavigationFocusableView` component receives the following props:
 
-[SpatialNavigationNode props](#props-1) except `isFocusable` that is already set to true.
-
-| Name        | Type                         | Default        | Description                                                                                                |
-| ----------- | ---------------------------- | -------------- | ---------------------------------------------------------------------------------------------------------- |
-| `style`     | `ViewStyle`                  | `null`         | Style for the View. This can be any valid React Native style
+| Name           | Type        | Default                 | Description                                                                                                                                                                  |
+| -------------- | ----------- | ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `style`        | `ViewStyle` | `null`                  | Style for the View. This can be any valid React Native style object.                                                                                                         |
+| `children`     | `ReactNode` | `null`                  | Child elements of the component. They are expected to be one or multiple `SpatialNavigationNode` elements.                                                                   |
+| `onMouseEnter` | `ReactNode` | `() => void \| boolean` | A function which is triggered when a pointer enters the `SpatialNavigationFocusableView`. To bypass the focus of the node when hovering it, the function must return `true`. |
+| `onMouseLeave` | `ReactNode` | `() => void`            | A function which is triggered when a pointer leaves the `SpatialNavigationFocusableView`.                                                                                    |
 
 ## Usage
 
 ```jsx
 <SpatialNavigationFocusableView
   onFocus={() => console.log('Node gained focus')}
   onSelect={() => console.log('Node was selected')}
+  onMouseEnter={() => console.log('The view has been entered by a pointer and the element has been focused'')}
+  onMouseLeave={() => console.log('The view has been left by a pointer')}
+  orientation="horizontal"
+>
+  {({ isFocused }) => <Text style={{ color: isFocused ? 'red' : 'black' }}>Hello World!</Text>}
+</SpatialNavigationFocusableView>
+```
+
+In this example above, the `onMouseEnter` function also triggers the focus of the element.
+
+```jsx
+<SpatialNavigationFocusableView
+  onFocus={() => console.log('Node gained focus')}
+  onSelect={() => console.log('Node was selected')}
+  onMouseEnter={() => {
+    console.log('The view has been entered by a pointer and the element has NOT been focused');
```

Comment:
I don't think the example should mention this case... keep it simple, and people with complex use cases will dive in and see if it's possible to not propagate the focus event, but I don't think this will ever happen!

Reply:
I deleted it 👍